### PR TITLE
Adds Hidden Polychromic Stuff to the Loadout

### PR DESCRIPTION
**Pull request by BurgerLUA**
[Changelogs]:
:cl: BurgerBB
add: Added polychromic cloaks, bottomless shirts, male and female tank tops, jumpsuits, pleated skirts, button-up shirts, PANTIES, shorts, capes, and skirts to the loadout.
tweak: Reduced the cost of the polychromic kilt from 3 to 2.
server: Some people may have to reapply their loadout to include already existing items, as the datums were renamed for consistency.
/:cl:

[why]: I noticed a lot of polychromic stuff missing in the loadout. This could be because of donator stuff as I noticed one of the capes was labeled "DONATOR" in a comment, but if not I think they should be added.

### CODE REVIEW

File: modular_citadel/code/modules/client/loadout/suit.dm [hunk 1]
```diff
@@ -80,6 +80,13 @@
 	path = /obj/item/clothing/suit/flakjack
 	cost = 2
 
+//Polychromic
+/datum/gear/polychromiccloak
```
Comment (Trilbyspaceclone) [inline modular_citadel/code/modules/client/loadout/suit.dm]: This is a donator item, prob low tear doator item but still a donator item...

Comment (BurgerLUA) [reply modular_citadel/code/modules/client/loadout/suit.dm]: Removed it.

**Review by silicons — APPROVED** [review]

**Review by silicons — APPROVED** [review]
Merging on approval from Jay/whoever did this

**Comment by LetterJay** [comment]
There's a high probability I'm going to remove this once the donator system is finished being revamped.

**Comment by Poojawa** [comment]
I think these need to be double checked for digitigrade compliance as well.

**Comment by silicons** [comment]
@LetterJay your call, closing this in 24h if it isn't merged.

**Pull request closed without merging**